data/versions.json

```diff
@@ -0,0 +1,53 @@
+{
+    "current": {
+        "name": "v6.3",
+        "date_start": "October 24, 2019",
+        "changelog": "updates-october-2019",
+        "cti_url": "https://github.com/mitre/cti/releases/tag/ATT%26CK-v6.3"
+    },
```

Comment:
We should stage this as a previous version for sub-techniques 2.1, and make the current version `ATT&CK-v7.0` (which doesn't technically exist yet, but will once we release the sub-techniques update). sub-techniques 2.1 isn't being released until the actual update on the 8th.

```
        {
            "name": "v6.3",
            "aliases": [],
            "date_start": "October 24, 2019",
            "date_end": "July 7, 2020",
            "changelog": "updates-october-2019",
            "cti_url": "https://github.com/mitre/cti/releases/tag/ATT%26CK-v6.3",
            "commit": "b15db5ce0b2f440cb690a3003ffc689098625d00"
        },
```

Please also modify the versions script remove the `/beta/` directory, or else we're going to preserve the sub-techniques beta with v6.3. We may have other betas in the future so this is actually useful long term.